addons/openebs/template/base/install.sh

```diff
@@ -48,12 +48,61 @@ function openebs() {
 function openebs_maybe_migrate_from_rook() {
     if [ -z "$ROOK_VERSION" ]; then
         if kubectl get ns | grep -q rook-ceph; then
+            # show validation errors from pvmigrate
+            # if there are errors, maybe_prompt_migrate_from_rook_dryrun_errors() will bail
+            maybe_prompt_migrate_from_rook_dryrun_errors
             rook_ceph_to_sc_migration "$OPENEBS_LOCALPV_STORAGE_CLASS"
             DID_MIGRATE_ROOK_PVCS=1 # used to automatically delete rook-ceph if object store data was also migrated
         fi
     fi
 }
 
+function maybe_prompt_migrate_from_rook_dryrun_errors() {
```

Comment:
This code is duplicated. Could we not centralize it in a common.sh or util.sh under the dir?

Reply:
the add-on version is generated from the template directory. it is duplicated in each add-on version. it is our pattern not to use shared code as we try not to change existing add-on versions to avoid breakage. this is likely a larger conversation if we want to change things at this level.